sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0-beta.4 (2022-10-11)
 
 ## Features Added
 - Added Upload method that takes in RequestContent and has GZip capability for efficiency
```

Comment:
Let's add a line item here for the new client registration extension methods you added